frontend/pages/admin/IntegrationsPage/cards/CertificateAuthorities/helpers.tsx

```diff
@@ -67,14 +82,15 @@ export const getCertificateAuthority = (
 
   if (id.includes("digicert")) {
     return digicertCerts?.find(
-      (cert) => id.split("digicert-")[1] === cert.id.toString()
-    );
+      (cert) => id.split("digicert-")[1] === cert.name
+    ) as ICertificatesIntegrationDigicert;
```

Comment:
Nit: I think this typecast can result in masking `undefined` as `ICertificatesIntegrationDigicert` to upstream callers.

Reply:
i have some new type guards that I can use here now. will make that change

Reply:
actually just adding another condition to the if statements will work